docs/setting-up/client/mongodb.md

```diff
@@ -33,140 +64,213 @@ db.getSiblingDB("admin").createRole({
 
-where username and password are credentials for the monitored MongoDB access, which will be used locally on the database host. Additionally, two positional arguments can be appended to the command line flags: a service name to be used by PMM, and a service address. If not specified, they are substituted automatically as `<node>-mongodb` and `127.0.0.1:27017`.
+> **See also**: [`pmm-admin` man page][PMM_ADMIN_MAN_PAGE]
 
-The command line and the output of this command may look as follows:
+**Example**
 
 ```sh
-pmm-admin add mongodb --username=pmm --password=pmm mongo 127.0.0.1:27017
+pmm-admin add mongodb \
+--username={{pmm_mongodb_user}} --password={{pmm_password}} \
+--query-source=profiler --cluster=mycluster
 ```
 
-```
-MongoDB Service added.
-Service ID  : /service_id/f1af8a88-5a95-4bf1-a646-0101f8a20791
-Service name: mongo
+**Example**
+
+```sh
+pmm-admin add mongodb \
+--username={{pmm_mongodb_user}} --password={{pmm_password}} \
+mongo 127.0.0.1:27017
 ```
 
-Beside positional arguments shown above you can specify service name and service address with the following flags: `--service-name`, `--host` (the hostname or IP address of the service), and `--port` (the port number of the service). If both flag and positional argument are present, flag gains higher priority. Here is the previous example modified to use these flags:
+**Example**
 
 ```sh
-pmm-admin add mongodb --username=pmm --password=pmm --service-name=mongo --host=127.0.0.1 --port=27017
+pmm-admin add mongodb \
+--username={{pmm_mongodb_user}} --password={{pmm_password}} \
+--service-name=mymongosvc --host=127.0.0.1 --port=27017
 ```
 
-You can add a MongoDB instance using a UNIX socket with the `--socket` option:
+**Example -- connect via UNIX socket**
 
 ```sh
 pmm-admin add mongodb --socket=/tmp/mongodb-27017.sock
 ```
 
-> If the password contains special symbols like the 'at' (`@`) symbol, the host might not be detected correctly. Make sure that you insert the password with special characters replaced with their escape sequences. The simplest way is to use the [`encodeURIComponent`][ENCODE_URI] JavaScript function in your browser's web console (usually found under *Development Tools*). Evaluate the function with your password as the parameter. For example:
->
-> ```javascript
-> encodeURIComponent('$ecRet_pas$w@rd')
-> "%24ecRet_pas%24w%40rd"
-> ```
 
-[ENCODE_URI]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent
+**Example -- connecting via SSL/TLS**
+
+```sh
+pmm-admin add mongodb --tls \
+--tls-certificate-key-file=PATHTOCER \
+--tls-certificate-key-file-password=IFPASSWORDTOCERTISSET \
+--tls-ca-file=PATHTOCACERT
+```
+
+where:
+
+- `PATHTOCERT`: Path to TLS certificate file.
+- `IFPASSWORDTOCERTISSET`: Password for TLS certificate file.
+- `PATHTOCACERT`: Path to certificate authority file.
+
+> **See also** [`pmm-admin` man page for `pmm-admin add mongodb`](../../details/commands/pmm-admin.md#mongodb)
```

Comment:
as an old unix guy 'man page' mean I could type 'man pmm-admin' and get help files on the CLI...we dont have this...I think you're trying to point people to the webpage but I don't know if calling it a man page would create confusion.  on CLI it's pmm-admin add mongodb --help or click the link so maybe replace man page with manual or online manual?

Reply:
As an even older UNIX guy I had created the pmm-admin 'man' page from `pmm-admin --help-man` which outputs troff. Some Unises' man command can take that and page it from stdin so that's an option but I've only tested it on Ubuntu (`pmm-admin --help-man | man -l -`)
I had intended to create a real man page via Sphinx. That's not possible with MkDocs/Markdown.